src/components/cargo.js

```diff
@@ -19,7 +26,7 @@ function Cargo(el, context) {
     const { cargo } = e.detail;
     const token = tokens[cargo.token] ? tokens[cargo.token].name : '-';
 
-    el.textContent = `Cargo: ${token}`;
+    cargoSlot.src = `../../assets/${token}.png`;
```

Comment:
Change all paths to assets to `/assets/...`, ie. an absolute path. Since this is handled by our HTTP-router, incase we would change the folder name or something else.

Reply:
Don't forget to test it also!

Reply:
Implemented and tested (it worked).